### ISSUE TITLE
Version 2.1.5 not listed on Papermc Hangar

### ISSUE THREAD

**Issue by Jakesta13** [MagicCheese1/Damage-Indicator#38]
Hello!
I use Papermc's API for grabbing latest releases of plugins from their plugin [listing site](https://hangar.papermc.io/Magiccheese1/DamageIndicator/versions) as grabbing direct URLs to the latest plugins a lot simpler this way.

I noticed that this plugin hasn't been updated on Papermc,
Not sure if Papermc grabs updates automatically and somehow missed this one?

**Comment by MagicCheese1** [comment]
No, this is done manually and just forgot about it. Thanks for the heads up! Will fix soon

**Comment by Jakesta13** [comment]
> No, this is done manually and I just forgot about it. Thanks for the heads up! Will fix soon

Much appreciated!
I wish github was easier to just **yoink** the latest file or at least list the files in the latest release for a api call to pull from, it seems to work in really specific cases if the specific filename never changes 🤷


Thanks again 😊

**Comment by MagicCheese1** [comment]
The newer version 2.2.0 is available on hangar so this issue can be closed
